evap/evaluation/views.py

```diff
@@ -161,17 +162,19 @@ def legal_notice(request):
 @require_POST
 @login_required
 def contact(request):
+    sent_anonymous = request.POST.get("anonymous") == "true"
+    if sent_anonymous and not settings.ALLOW_ANONYMOUS_FEEDBACK_MESSAGES:
+        raise SuspiciousOperation("Anonymous feedback messages are not allowed, however received one from user!")
     message = request.POST.get("message")
     title = request.POST.get("title")
-    email = request.user.email or f"User {request.user.id}"
+    email = settings.DEFAULT_FROM_EMAIL if sent_anonymous else request.user.email or f"User {request.user.id}"
     subject = f"[EvaP] Message from {email}"
```

Comment:
This would mean that for anonymous messages, the subject would be:
"[EvaP] Message from evap@hpi.de"

-- @janno42 does this match what we want to happen?

Reply:
The subject should be "[EvaP] Anonymous message" and the `reply_to` header should be empty.